src/contributing/packaging.md

```diff
@@ -7,27 +7,31 @@ To package Darling for Debian-based systems, we provide the `tools/debian/makede
 All output files are stored in the parent directory of the source root because of a technical limitation of `debuild`.
 
 ### Install Dependencies
-```bash
-$ sudo apt install devscripts equivs dpkg-dev debhelper
+```
+bash
```

Comment:
I changed this because many of the other code segments start with a new line - I was trying to make it consistent. Are you sure you want me to change these back?

Reply:
> I changed this because many of the other code segments start with a new line - I was trying to make it consistent.

Markdown supports a feature called [Syntax highlighting](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks#syntax-highlighting).

---

What your change did was change this
```bash
$ sudo apt install devscripts equivs dpkg-dev debhelper
```
into this
```
bash
sudo apt install devscripts equivs dpkg-dev debhelper
```

It doesn't really make sense for the user to type in `bash` before they run the `sudo apt install` command.